### ISSUE TITLE
Small display bug in LHS menu

### ISSUE THREAD

**Issue by lurch** [raspberrypi/documentation#2595]
If I'm currently looking at https://www.raspberrypi.com/documentation/microcontrollers/c_sdk.html then the LHS menu displays correctly:
![Screenshot from 2022-07-19 12-57-25](https://user-images.githubusercontent.com/476186/179744626-eb86aa16-1148-4d6f-9a10-23b0e191f89d.png)

And if I'm looking at https://www.raspberrypi.com/documentation/microcontrollers/micropython.html then the menu also displays correctly:
![Screenshot from 2022-07-19 12-59-09](https://user-images.githubusercontent.com/476186/179744941-61d11133-c742-46fa-a0e2-f64639a1718f.png)

...but if I _then_ click on the "**The C/C++ SDK**" header to expand just that sub-section of the menu, then one of the sub-menu-items displays "Raspberry Pi Pico C/{cpp} SDK" instead of "Raspberry Pi Pico C/C++ SDK":
![Screenshot from 2022-07-19 13-06-49](https://user-images.githubusercontent.com/476186/179746450-25abaa5e-8cf0-4466-aef6-507a7cf05b2d.png)

**Comment by aallan** [comment]
Hmm. I can't reproduce this in Chrome on macOS. Interesting.

**Comment by lurch** [comment]
To clarify, in that last screenshot I'm still on the https://www.raspberrypi.com/documentation/microcontrollers/micropython.html page and all I've done is expanded part of the menu (by clicking on the bold header); I've not navigated to a different page (by clicking on one of the non-bold headers).

**Comment by NachiThePanda** [comment]
@aallan Im running Windows 11 using the Latest version of Opera GX

I checked what @lurch was saying and I I'm getting the same issue. Surprisingly, [dev tools shows this](https://ibb.co/vj342ZL) so it could be how the browser is handling the text or character placement?

**Comment by lurch** [comment]
Yeah, it's a minor bug in our build system, which isn't important enough to warrant fixing yet :slightly_smiling_face:

**Comment by katshann-raspberry** [comment]
@lurch  I can't reproduce. Looks like this may have been fixed in the last 2 years. If you are happy that it's fixed, can I close it?

**Comment by lurch** [comment]
Yeah, looks like it was probably fixed by some of the restructuring that Nate did.
